### ISSUE TITLE
Window_Control Failed unsupported_command

### ISSUE THREAD

**Issue by venturanc** [israndy/TesSense#4]
Thank you so much for sharing this!

I have noticed one issue though, which is an error message of "Window_Control Failed unsupported_command", which happens when calling the (vent? close?) window commands as of March 2023 (*possibly as early as January 2023?). 

I found a few references online that suggested this is due to Tesla removing the function: https://www.notateslaapp.com/news/1136/tesla-to-remove-window-auto-close-features-due-to-new-regulations

-----
Background:
I don't have any of the Sense or TPLink stuff, but I have a Tesla Powerwall 2/Gateway, and am running the jasonacox/pypowerwall (https://github.com/jasonacox/pypowerwall/) to easily access the instant grid/solar/home/battery power usage from the Powerwall Gateway over the local network (and make that data accessible to SmartThings home automation).

I was able to modify your code and get it working with the local gateway data instead of the Sense monitor, and set it to automatically stop charging the car at 4pm (to avoid draining the Powerwall or using peak grid power from 4-9PM). (After I clean it up, I would like to get it posted to GitHub- pypowerwall runs as a docker container, and I was thinking that it would be really useful to figure out how to package my updated script to run in the pypowerwall docker, and accept the email/lat/long/night-time hours, and teslapy token as environmental variables, with a web api route to turn on and off the auto-solar-charging in case someone wanted to disable or turn it on it with a simple GET call to an endpoint on the pypowerwall webserver ).

**Comment by israndy** [comment]
Yes, I heard the vent command was going away, but it’s not warm enough here yet to have the command issued, I guess I’ll pull it, but I haven’t really had any updates in a while. Just trying to track down all the bugs that cause it to not make it thru the day w/o a fail.

What you are planning sounds very interesting, but I am not familiar how to help you with it.
